README.md

```diff
@@ -35,7 +35,10 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
 * There must be at least one space around operators, and (except for aligning multiline statements) at most one, see the
   [respective Squizlabs Sniff](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#squizwhitespaceoperatorspacing)
   we imported with `ignoreNewlines = false`
-* Single quotes must be used instead of double quoutes, where possible.
+* Single quotes must be used instead of double quotes, where possible.
+* all method requires a return type; use `void` if the method returns nothing.
```

Comment:
"all methods require"

Reply:
And for consistency reasons with the bullet points above I'd vote for uppercase first :)